provider/gce/environ_availzones.go

```diff
@@ -54,6 +54,18 @@ func (env *environ) InstanceAvailabilityZoneNames(ids []instance.Id) ([]string,
 	return results, err
 }
 
+// DeriveAvailabilityZone is part of the common.ZonedEnviron interface.
+func (env *environ) DeriveAvailabilityZone(args environs.StartInstanceParams) (string, error) {
+	// ToDo (HML) 16-Oct-2017
```

Comment:
nit: "TODO" is more conventional, please stick to that (here and elsewhere)

Reply:
done